mysqlMasker_test.go

```diff
@@ -1,12 +1,12 @@
-package parser
+package masker
 
 import (
 	"strings"
 	"testing"
 )
 
-func TestPerseValidQuery(t *testing.T) {
-	querys := []string{
+func TestMaskValidMysqlQuery(t *testing.T) {
+	queries := []string{
```

Comment:
I suggest Go test libraly [gotest](https://github.com/cweill/gotests).
it is generate table test case.

Reply:
9296b41 , 391e4d0
I fix test according to gotest(table driven test).  Thank you for your suggestion.